.github/workflows/checklist.yml

```diff
@@ -0,0 +1,22 @@
+name: PR Checklist
+
+on:
+  pull_request_target:
```

Comment:
I'm hoping this change fixes the issue with community PRs

Reply:
what's the issue?

Reply:
This jobs keeps failing for community PR. [example](https://github.com/n8n-io/n8n/actions/runs/5530465399).